Dockerfile

```diff
@@ -18,10 +18,11 @@ RUN go mod download
 FROM build_base AS server_builder
 ARG TARGETARCH
 ARG GITHASH="unknown"
+ARG WEAVIATE_VERSION="unknown"
 ARG EXTRA_BUILD_ARGS=""
 COPY . .
 RUN GOOS=linux GOARCH=$TARGETARCH go build $EXTRA_BUILD_ARGS \
-      -ldflags '-w -extldflags "-static" -X github.com/weaviate/weaviate/usecases/config.GitHash='"$GITHASH"'' \
+      -ldflags '-w -extldflags "-static" -X github.com/weaviate/weaviate/usecases/config.GitHash='"$GITHASH"' -X github.com/weaviate/weaviate/usecases/config.WeaviateVersion='"$WEAVIATE_VERSION"'' \
```

Comment:
We do already have a way of getting Weaviate version, look [here](https://github.com/weaviate/weaviate/blob/stable/v1.24/adapters/handlers/rest/configure_api.go#L410) you don't have to pass an extra param to the Dockerfile